iceoryx_binding_c/cmake/iceoryx_binding_c_deployment.h.in

```diff
@@ -16,6 +16,7 @@
 #ifndef IOX_BINDING_C_DEPLOYMENT_H
 #define IOX_BINDING_C_DEPLOYMENT_H
 
-#define IOX_BUILD_GENERATED_MAX_NUMBER_OF_NOTIFIERS @IOX_MAX_NUMBER_OF_NOTIFIERS@
+/// @note must be equal to MAX_NUMBER_OF_NOTIFIERS in iceoryx_posh_types.hpp
```

Comment:
Could you please also add a hint here which files one has to touch when this is going to be activated.
* `iceoryx_posh/cmake/IceoryxPoshDeployment.cmake`
* `iceoryx_posh/cmake/iceoryx_posh_deployment.hpp.in `
* `iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp`

Reply:
Comment added